class/defaults.yml

```diff
@@ -24,6 +24,7 @@ parameters:
         class: ssd
         fstype: ext4
         volumeexpansion: true
+        retainpolicy: Retain
```

Comment:
Would keep the default Delete in the component. We might change that for certain customers.